docs/sql/settings.md

```diff
@@ -295,3 +295,240 @@ Result set:
   }
 }
 ```
+
+## opendistro.sql.query.response.format
+
+### Description
+
+You can set default response format of the query. The supported formats include: jdbc, json, csv, raw, and table.
+
+1. The default value is jdbc.
+2. This setting is node scope.
+3. This setting can be updated dynamically.
+
+
+### Example 1
+
+You can update the setting with a new value like this.
+
+SQL query:
+
+```console
+>> curl -H 'Content-Type: application/json' -X PUT localhost:9200/_opendistro/_sql/settings -d '{
+  "transient" : {
+    "opendistro.sql.query.response.format" : "json"
+  }
+}'
+```
+
+Result set:
+
+```json
+{
+  "acknowledged" : true,
+  "persistent" : { },
+  "transient" : {
+    "opendistro" : {
+      "sql" : {
+        "query" : {
+          "response" : {
+            "format" : "json"
+          }
+        }
+      }
+    }
+  }
+}
+```
+
+### Example 2
+
+After the setting is updated, query result is like:
+
+SQL query:
+
+```console
+>> curl -H 'Content-Type: application/json' -X POST localhost:9200/_opendistro/_sql -d '{
+  "query" : "SELECT firstname, lastname, age FROM accounts ORDER BY age LIMIT 2"
+}'
+```
+
+Result set:
+
+```json
+{
+  "_shards" : {
+    "total" : 5,
+    "failed" : 0,
+    "successful" : 5,
+    "skipped" : 0
+  },
+  "hits" : {
+    "hits" : [
+      {
+        "_index" : "accounts",
+        "_type" : "_doc",
+        "_source" : {
+          "firstname" : "Nanette",
+          "age" : 28,
+          "lastname" : "Bates"
+        },
+        "_id" : "13",
+        "sort" : [
+          28
+        ],
+        "_score" : null
+      },
+      {
+        "_index" : "accounts",
+        "_type" : "_doc",
+        "_source" : {
+          "firstname" : "Amber",
+          "age" : 32,
+          "lastname" : "Duke"
+        },
+        "_id" : "1",
+        "sort" : [
+          32
+        ],
+        "_score" : null
+      }
+    ],
+    "total" : {
+      "value" : 4,
+      "relation" : "eq"
+    },
+    "max_score" : null
+  },
+  "took" : 100,
+  "timed_out" : false
+}
+```
+
+## opendistro.sql.cursor.enabled
+
+### Description
+
+You can enable or disable pagination for all queries that are supported.
+
+1.  The default value is false.
+2.  This setting is node scope.
+3.  This setting can be updated dynamically.
+
+### Example
+
+You can update the setting with a new value like this.
+
+SQL query:
```

Comment:
Feels unnecessary given that it's the same exact call as earlier with a slightly different payload.